source/ui/Popup.js

```diff
@@ -219,6 +219,7 @@
 		* @private
 		*/
 		handlers: {
+			tap: 'tap',
```

Comment:
I don't think we need to explicitly declare this as we call `tap` for tap events if a handler with this name exists. Alternatively, if we wanted to be explicit, then this should be `ontap`.